examples/nextjs/src/components/shared/contentlets.js

```diff
@@ -1,54 +1,60 @@
-"use client";
-import React from "react";
-import Image from "next/image";
-import Contentlet from "./contentlet";
+'use client';
+import React from 'react';
+import Image from 'next/image';
+import { editContentlet } from '@dotcms/client';
 
 const dateFormatOptions = {
-    year: "numeric",
-    month: "long",
-    day: "numeric",
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric'
 };
 
 function Contentlets({ contentlets }) {
     return (
         <ul className="flex flex-col gap-7">
             {contentlets.map((contentlet) => (
-                <Contentlet contentlet={contentlet} key={contentlet.identifier}>
-                    <li className="flex gap-7 min-h-16">
+                <li className="flex gap-7 min-h-16 relative" key={contentlet.identifier}>
+                    <button
+                        onClick={() => editContentlet(contentlet)}
+                        style={{
+                            color: 'black',
+                            border: '1px solid black',
+                            position: 'absolute',
+                            bottom: 0,
+                            right: 0,
+                            width: '40px',
+                            height: '20px',
+                            zIndex: 1000,
+                            display: 'flex',
+                            justifyContent: 'center',
+                            alignItems: 'center'
+                        }}>
```

Comment:
Custom `CSS` class or tailwind? Make it easier to read for external devs

Reply:
This will be a component. I hardcoded it for testing purposes